### PR TITLE
chore: bump amplify codegen to 3.3.3

### PR DESCRIPTION
**Pull request by AaronZyLee**
#### Description of changes
Add hardcoded modelName change for dart modelgen.
Full changelog: https://github.com/aws-amplify/amplify-codegen/releases/tag/amplify-codegen%403.3.3

#### Checklist

- [x] PR description included
- [x] `yarn test` passes
- [x] Tests are [changed or added](https://github.com/aws-amplify/amplify-cli/blob/dev/CONTRIBUTING.md#tests)
- [ ] Relevant documentation is changed or added (and PR referenced)
- [ ] New AWS SDK calls or CloudFormation actions have been added to relevant test and service IAM policies
- [ ] [Pull request labels](https://github.com/aws-amplify/amplify-cli/blob/dev/CONTRIBUTING.md#labels) are added

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by evcodes — APPROVED** [review]
LGTM

**Review by alharris-at — DISMISSED** [review]

**Review by danielleadams — APPROVED** [review]

**Comment by danielleadams** [comment]
Just started e2e tests, once those pass, this is good to merge.

**Review by jhockett — APPROVED** [review]

**Review by danielleadams — APPROVED** [review]
LGTM pending e2e tests.

**Review by alharris-at — APPROVED** [review]

**Pull request merged**
